libs/pilight/protocols/433.92/nexus.c

```diff
@@ -124,7 +124,7 @@ static void parseCode(char **message) {
             // store or apply the settings
             temperature += settings->temperature_offset;
             humidity += settings->humidity_offset;
-            temperature_decimals = settings->temperature_decimals;
+            temperature_decimals = (int)round(settings->temperature_decimals);
```

Comment:
But casting to `int` here. Why?

Reply:
In the initial PR (not rewrite) this was a double. I changed it to int, but had a compiler warning because json_find_number seems to return a double, so I reverted it. However, I need an int, since that's what makes sense and that's what the printf %.*f needs as an argument for the number of decimals.
I've looked at other protocols and did not see what they do with this number. So I left it as a double but made it actually do something.

Reply:
True